src/renderer/gl_engine/tvgGlRenderPass.h

```diff
@@ -74,6 +74,10 @@ class GlRenderPass
     }
 
     int nextDrawDepth() { return ++mDrawDepth; }
+
+    void setDrawDepth(int32_t depth) { mDrawDepth = depth; }
+
```

Comment:
What is global purpose of the depth buffer in rendering pipelines in gl_engine?

Reply:
It is used for path clip.